features/step_definitions/machine.rb

```diff
@@ -63,7 +63,7 @@
   teardown_add {
     num_actual = 0
 
-    success = wait_for(600, interval: 30) {
+    success = wait_for(1200, interval: 20) {
```

Comment:
@jhou1 IMO, 20 minutes is big long...it might be a OCP issue if we have to wait for that long

Reply:
Yes, normally in my test, the test could properly clean up and finish. It is in the ci, that the cluster didn't scale down successfully in test tear down and I can not reproduce it locally.

```
Failed to restore cluster, expected number of nodes 6, got 7 (RuntimeError)
```

Scale down takes longer time because there is a node drain logic during scale down and the drain must be successful. If there are more nodes to scale down, it probably increases the wait time. My wait time is too aggressive, I'll try to increase to 900 and test on more clusters to see if this fixes the test flake.